### DISCUSSION TITLE
Tramming does not agree with mesh?

### DISCUSSION THREAD

**Discussion by cds333** [mriscoc/Ender3V2S1#263] [Q&A]
Or perhaps I should say "mesh doesn't agree with tramming"...

Why is it that I can manually tram the bed by going from corner to corner and adjusting each one until they all say 0.00 (which sometimes takes multiple times around the bed), and then without heating the bed or literally doing anything else (I didn't move the printer, touch anything, bump the bed, absolutely nothing) I go to auto build mesh (set at 0C) and get something like this:

![image](https://user-images.githubusercontent.com/16358097/182012899-0c2b620f-48bf-4a69-aab5-efba07da7be2.png)

Shouldn't the corners I just probed while tramming at least show the same values?

The bed wheels are not loose, the bed is not warped, the glass plate is secured in place tightly, the printer is on a level surface...

Tried this with both BL and CRTouch btw.

Please advise, thanks.

**Comment by mriscoc** [comment]
Yes, if you have a good bed tramming then the same points in the mesh will be in tolerance (0.05 mm). 
There is a reported problem with some S1 boards, in those cases is better use a mesh of 5x5 or smaller and correct the leveling values manually using the mesh editor.

**Comment by albertoabrao** [comment]
I've tried a lot of things on my S1 (F4 board).

So far, the most consistent results are by doing the following:

- Always use bed tramming/mesh leveling with the bed at room temperature.

- Taken from Creality's firmware for the S1 F4:
Recv: echo:; Maximum feedrates (units/s):
Recv: echo:  M203 X300.00 Y300.00 Z5.00 E25.00

The Z value is different, I think the Professional Frimware uses 20 or so. It makes a *HUGE* difference.

I've been using all the default settings for mesh probing otherwise, and so far so good.

Try it. I would appreciate if you could report back with your results. Hope it helps.
